technical-guide/developer/ui.md

```diff
@@ -660,3 +660,84 @@ You can use Storybook's `<Canvas>` element to showcase specific stories to enric
 When including codeblocks, please add code in Clojure syntax (not JSX).
 
 You can find an example MDX file in the [Buttons docs](https://hourly.penpot.dev/storybook/?path=/docs/buttons-docs--docs).
+
+### Replacing and old component
+
+**Create screenshots for visual tests BEFORE making any changes.**
```

Comment:
We need to clarify what to do if the new component has had their design altered (like what happened when you subbed the previous tabs component for the new one)

Reply:
Solved at the end